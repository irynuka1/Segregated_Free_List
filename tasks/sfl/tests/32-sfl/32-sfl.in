INIT_HEAP 0x1 2 16 1
MALLOC 8
MALLOC 8
MALLOC 6
MALLOC 5
MALLOC 1
DUMP_MEMORY
FREE 0x11
FREE 0x17
FREE 0x1c
FREE 0x9
FREE 0x1
DUMP_MEMORY
DESTROY_HEAP
