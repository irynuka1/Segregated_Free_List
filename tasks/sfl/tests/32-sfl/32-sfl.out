+++++DUMP+++++
Total memory: 32 bytes
Total allocated memory: 28 bytes
Total free memory: 4 bytes
Free blocks: 1
Number of allocated blocks: 5
Number of malloc calls: 5
Number of fragmentations: 3
Number of free calls: 0
Blocks with 4 bytes - 1 free block(s) : 0x1d
Allocated blocks : (0x1 - 8) (0x9 - 8) (0x11 - 6) (0x17 - 5) (0x1c - 1)
-----DUMP-----
+++++DUMP+++++
Total memory: 32 bytes
Total allocated memory: 0 bytes
Total free memory: 32 bytes
Free blocks: 6
Number of allocated blocks: 0
Number of malloc calls: 5
Number of fragmentations: 3
Number of free calls: 5
Blocks with 1 bytes - 1 free block(s) : 0x1c
Blocks with 4 bytes - 1 free block(s) : 0x1d
Blocks with 5 bytes - 1 free block(s) : 0x17
Blocks with 6 bytes - 1 free block(s) : 0x11
Blocks with 8 bytes - 2 free block(s) : 0x1 0x9
Allocated blocks :
-----DUMP-----
